tests/unit/authenticators/ucentral-router-test.js

```diff
@@ -0,0 +1,92 @@
+import { module, test } from "qunit";
+import { setupMirage } from "ember-cli-mirage/test-support";
+import UcentralRouterAuthenticator from "ucentral/authenticators/ucentral-router";
+import { setupTest } from "ember-qunit";
+import { Response } from "miragejs";
+
+module("Unit | Authenticator | UcentralRouterAuthenticator", function (hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    this.authenticator = UcentralRouterAuthenticator.create();
```

Comment:
```suggestion
    this.authenticator = this.owner.lookup("authenticator:ucentral-router");
```

Instances of anything should always be resolved via the container.